Directory.Build.props

```diff
@@ -26,4 +26,10 @@
     </ItemGroup>
   </Target>
 
+  <ItemGroup>
```

Comment:
This should be removed since not part of the PR :)

Reply:
Good point, fixed now